portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Olu's Portfolio">
+    <meta name="keywords" content="HTML, CSS, JavaScript, Olu, Portfolio, CS">
+    <meta name="author" content="Olu Alamutu">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
+    <link rel="manifest" href="images/site.webmanifest">
+  </head>
+  <body>
+    <div class="row">
+      <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+      <li><a href="index.html" style="color:white"> HOME </a></li>
+      <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+	  <li><a href="showcase.html" style="color:white"> PROJECT SHOWCASE </a></li>
+      <li><a href="about.html" style="color:white"> ABOUT </a></li>
+      <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+      <li><a href="stock.html" style="color:white"> STOCK PICKER</a></li>
+      <li class="active"><a href="comments.html" style="color:white"> COMMENTS</a></li>
+    </ul>
+    <h1 style=margin-top:50px>&#128172<i>Feel Free to Leave Comments</i>&#128172</h1>
+    <form method="POST" action="/data">
+      <input type="text" name="comment"/>
+      <br/>
+      <button>Submit</button>
+    </form>
+    <label for="useroptions" style="background:gray">How many comments would you like to see?:</label>
+    <select name="useroptions" id="maxwant">
+      <option value="1">One</option>
+      <option value="2">Two</option>
+      <option value="3">Three</option>
+      <option value="4">Four</option>
+      <option value="5">Five</option>
+      <option value="10">Ten</option>
+      <option value="20">Twenty</option>
+      <option value="30">Thirty</option>
+      <option value="40">Fourty</option>
+      <option value="50">Fifty</option>
+      <option value="100">One-Hundred</option>
+      <option value="200">Two-Hundred</option>
+      <option value="99999">All</option>
+    </select>
+    <div id="content"></div>
+        <button onclick="deleteComment()" style="background-color:red">Click here to delete comments</button>
+    <div id="comment-list"></div>
+    <div id="content">
```

Comment:
I see that we repeated `<div id="content">` here and on line 52. Was that intentional? If so, let's change that to `<div class="content">`. Typically, IDs should only be used for a single element.

Reply:
Addressed in latest commit.